src/view/com/util/UserAvatar.tsx

```diff
@@ -50,8 +50,9 @@ interface EditableUserAvatarProps extends BaseUserAvatarProps {
 
 interface PreviewableUserAvatarProps extends BaseUserAvatarProps {
   moderation?: ModerationUI
-  onBeforePress?: () => void
   profile: AppBskyActorDefs.ProfileViewBasic
+  disable?: boolean
```

Comment:
Let's try to be a little more descriptive with naming e.g. `disableHoverCard`, but nbd

Reply:
threw it in another pr im merging 👍